website/docs/r/logging_project_sink.html.markdown

```diff
@@ -0,0 +1,120 @@
+---
+layout: "google"
+page_title: "Google: google_logging_project_sink"
+sidebar_current: "docs-google-logging-project-sink"
+description: |-
+  Manages a project-level logging sink.
+---
+
+# google\_logging\_project\_sink
+
+Manages a project-level logging sink. For more information see
+[the official documentation](https://cloud.google.com/logging/docs/),
+[Exporting Logs in the API](https://cloud.google.com/logging/docs/api/tasks/exporting-logs)
+and
+[API](https://cloud.google.com/compute/docs/reference/latest/instances).
+
+Note that you must have the "Logs Configuration Writer" IAM role (`roles/logging.configWriter`)
+granted to the credentials used with terraform.
+
+## Example Usage
+
+```hcl
+resource "google_logging_project_sink" "my-sink" {
+    name = "my-pubsub-instance-sink"
+
+    // Can export to pubsub, cloud storage, or bigtable
```

Comment:
https://www.terraform.io/docs/configuration/syntax.html
> Single line comments start with #

Reply:
Hah apparently it also accepts // as i've been using them in my hcl unknowingly for a while!

Changed to #